updatehub/src/main.rs

```diff
@@ -114,7 +114,31 @@ async fn client_main(cmd: ClientCommands) -> updatehub::Result<()> {
 
     match cmd {
         ClientCommands::Info(_) => println!("{:#?}", client.info().await),
-        ClientCommands::Log(_) => println!("{:#?}", client.log().await),
+        ClientCommands::Log(_) => {
+            for entry in client.log().await?.into_iter() {
+                let level = match entry.level {
+                    sdk::api::log::Level::Critical => "CRIT",
+                    sdk::api::log::Level::Error => "ERRO",
+                    sdk::api::log::Level::Warning => "WARN",
+                    sdk::api::log::Level::Info => "INFO",
+                    sdk::api::log::Level::Debug => "DEBG",
+                    sdk::api::log::Level::Trace => "TRCE",
+                };
+
+                println!(
+                    "[{timestamp}] {level} {msg}",
+                    timestamp = entry
+                        .time
+                        .chars()
+                        .take(19)
+                        .collect::<Vec<_>>()
+                        .into_iter()
+                        .collect::<String>(),
```

Comment:
Output still doesn't match the expected pattern, heres an example of the log output:

```
Aug 11 11:54:06.322 DEBG delaying 86399 seconds till next probe
Aug 11 11:54:06.322 TRCE delaying transition for: 86399 seconds
Aug 11 11:54:16.199 DEBG receiving log request
```

And this is the current format from the client log command:
```
[2020-08-11 11:54:06] DEBG delaying 86399 seconds till next probe
[2020-08-11 11:54:06] TRCE delaying transition for: 86399 seconds
[2020-08-11 11:54:16] DEBG receiving log request
```

We should have these two matching.